LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -1978,12 +1978,12 @@ TEMPLAR_PISTOL_RANGE[25] = -100
 
 [LW_WeaponsAndArmor.X2Ability_FactionWeaponAbilities]
 // ***** Mobility bonuses for Bullpups
-BULLPUP_CONVENTIONAL_MOBILITY_BONUS = 3
-BULLPUP_MAGNETIC_MOBILITY_BONUS = 3
-BULLPUP_BEAM_MOBILITY_BONUS = 3
+BULLPUP_CONVENTIONAL_MOBILITY_BONUS = 2
+BULLPUP_MAGNETIC_MOBILITY_BONUS = 2
+BULLPUP_BEAM_MOBILITY_BONUS = 2
```

Comment:
Not quite a cleanup of unneeded files :wink:

Reply:
Yeah that got included in part of that commit. I forgot to mention that change as part of that commit. Unless you meant something else.

Reply:
It's better to separate independent, unrelated changes into different commits. I can probably clean this up when I merge.

Reply:
ahh ok gotcha. I'll make sure to make unrelated changes separate commits.